go.mod

```diff
@@ -16,9 +17,10 @@ require (
 	k8s.io/cli-runtime v0.18.8
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/code-generator v0.18.8
-	knative.dev/eventing v0.18.0
+	knative.dev/eventing v0.18.5-0.20201105155307-650096a39064
+	knative.dev/kn-plugin-source-kafka v0.0.0-20201203183209-e1f755efaca3
```

Comment:
we'll get the release done tomorrow to have this pinned at a tag

Reply:
Yep, it's pulling `master` version directly. I'm expecting some nits in Kafka setup, therefore though might be a good idea to give a head start.